File: iniciante/javascript/1165.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let testes = parseInt(lines.shift())

while(testes--){
    let numero = parseInt(lines.shift())
    let count = 1
    for(let i = 2; i <= numero; i++){
        if(numero % i === 0)
            count++
        if(count > 2){
            console.log(`${numero} nao eh primo`)
            break
        }    
    }
    if(count == 2)
        console.log(`${numero} eh primo`)
}

File: iniciante/python/1165.py
# -*- coding: utf-8 -*-

count_entrada = 0
count_divisores = 0

qtd_entrada = int(input())
while count_entrada < qtd_entrada:
    entrada = int(input())
    
    for i in range(1, entrada + 1):
        if entrada % i == 0:
            count_divisores += 1
        if count_divisores > 2:
            break
    if count_divisores > 2:
        print(f'{entrada} nao eh primo')
    else:
        print(f'{entrada} eh primo')

    count_divisores = 0
    count_entrada += 1
